### PR TITLE
(CORE only changes) Kerkko/lab 1207 click aliastemplate uses wrong

### PR DESCRIPTION
**Pull request by xylix**

### CODE REVIEW

File: packages/core/src/RPA/core/geometry.py [hunk 1]
```diff
@@ -80,9 +85,32 @@ def center(self):
     def as_tuple(self):
         return astuple(self)
 
+    def scale(self, scaling_factor: int):
```
[the remark this answers is not in the document]
Comment (osrjv) [reply packages/core/src/RPA/core/geometry.py:88]: as long as it rounds them back to ints, I guess.

File: packages/core/src/RPA/core/geometry.py [hunk 2]
```diff
@@ -59,6 +60,10 @@ def __iter__(self):
     def from_size(cls, x, y, width, height):
         return cls(x, y, x + width, y + height)
 
+    @classmethod
+    def from_mss_dict(cls, left, top, width, height):
```
[the remark this answers is not in the document]
Comment (osrjv) [reply packages/core/src/RPA/core/geometry.py:64]: and core doesn't know anything about mss. have it as some static helper inside the desktop library instead?

File: packages/core/src/RPA/core/geometry.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (xylix) [reply packages/core/src/RPA/core/geometry.py:88]: Oh right 0.5 definitely is not an int. Will fix

File: packages/core/src/RPA/core/geometry.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (xylix) [reply packages/core/src/RPA/core/geometry.py:64]: I think isolating the names of coordinate properties like "left top width height" into one place (so here geometry.py) makes most sense. Changing it to in a dict is also an option.

```
screenshot = self.ctx.take_screenshot(
    locator=Region.from_mss_dict(**display)
)
```
The usage for this feature will basically look like this. Also with this new classmethod Get Display Dimensions can also be a bit nicer.

Current implementation:
```
    @keyword
    def get_display_dimensions(self) -> Region:
        """ comment omitted for brevity """
        with mss.mss() as sct:
            disp = sct.monitors[0]
            return Region.from_size(
                disp["left"], disp["top"], disp["width"], disp["height"]
            )
```

[the remark this answers is not in the document]
Comment (xylix) [reply packages/core/src/RPA/core/geometry.py:64]: Sure I can change it to a static helper anyway. Github didn't load that comment at first 🤔

[the remark this answers is not in the document]
Comment (osrjv) [reply packages/core/src/RPA/core/geometry.py]: it's not a big problem, just a weird-ish semi-dependency to mss. we could either rename the existing function's arguments to left/top (would be more inline with the rest of the dataclass but backwards incompatible), or call it something like `from_display` and have it explicitly take the values from the dict.

**Pull request merged**